parsing/dialects/intermediate/dialects/dialect1/parserImpls.ftl

```diff
@@ -5556,3 +5559,17 @@ SqlIterateStmt IterateStmt() :
     <ITERATE> label = SimpleIdentifier()
     { return new SqlIterateStmt(getPos(), label); }
 }
+
+SqlSetStmt SetStmt() :
+{
+    final SqlIdentifier target;
+    final SqlNode source;
+    final Span s = Span.of();
+}
+{
+    <SET>
+    target = SimpleIdentifier()
```

Comment:
Can target be a CompoundIdentifier()?

Reply:
It cannot be a CompoundIdentifier(), since it is supposed to be either a parameter or a local variable of the store procedure .